templates/components/engines/python/agent/tools/interpreter.py

```diff
@@ -14,14 +14,15 @@
 
 class InterpreterExtraResult(BaseModel):
     type: str
-    filename: str
-    url: str
+    content: Optional[str] = None
```

Comment:
The e2b result is not only a file but also a raw text, html markdown text.

Reply:
@leehuwuj is that not independant of using files?